### PR TITLE
fix: delete running docker containers and fix counter test

### PR DESCRIPTION
**Pull request by cdurai-netapp**
1. increase timeouts
2. remove used/unused containers/images and volumes

### CODE REVIEW

File: integration/Jenkinsfile [hunk 1]
```diff
@@ -180,8 +180,9 @@ pipeline {
 
 def void stopAndRemoveDockers() {
     sh '''
-        docker system prune --all --force --volumes
-
+        for container_id in $(docker ps -a -q);do docker stop $container_id && docker rm -f $container_id;done
+        for image_id in $(docker images -q);do docker rmi -f $image_id;done
```
Comment (rahulguptajss) [inline integration/Jenkinsfile:184]: docker rmi can take list also. May be try docker rmi -f $(docker images -a -q)

File: integration/Jenkinsfile [hunk 2]
```diff
@@ -180,8 +180,9 @@ pipeline {
 
 def void stopAndRemoveDockers() {
     sh '''
-        docker system prune --all --force --volumes
-
+        for container_id in $(docker ps -a -q);do docker stop $container_id && docker rm -f $container_id;done
```
Comment (rahulguptajss) [inline integration/Jenkinsfile:183]: isn't this equal to docker container prune?

File: integration/Jenkinsfile [hunk 3]
```diff
@@ -180,8 +180,9 @@ pipeline {
...
+        for image_id in $(docker images -q);do docker rmi -f $image_id;done
+        for volume_id in $(docker volume ls -q); do docker volume rm -f $volume_id;done
```
Comment (rahulguptajss) [inline integration/Jenkinsfile:185]: isn't this equal to docker volume prune?

File: integration/Jenkinsfile (hunk 2 above)
Comment (cdurai-netapp) [reply integration/Jenkinsfile]: fixed

File: integration/Jenkinsfile (hunk 3 above)
Comment (cdurai-netapp) [reply integration/Jenkinsfile]: addressed

**Review by cgrinds — APPROVED** [review]

**Review by rahulguptajss — APPROVED** [review]

**Pull request merged**